src/main/java/com/google/sps/servlets/DeleteRecipeServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for deleting recipes. */
+@WebServlet("/delete-recipe")
+public class DeleteRecipeServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long id = Long.parseLong(request.getParameter("id"));
+
+    Key taskEntityKey = KeyFactory.createKey("PrivateRecipe", id);
```

Comment:
Can only private recipes be deleted? If so, maybe add a comment here with a little explanation on why you made that design decision

Reply:
right now private recipes are user uploaded recipes; public recipes right now are already part of the website, no one can delete them
eventually we'll let users publish their recipes and make them public but that's for after the MVP